lua/neogit/lib/git/cli.lua

```diff
@@ -443,7 +452,8 @@ local cli = setmetatable({
     local calls = {...}
     return new_parallel_builder(calls)
   end,
-  git_root = git_root
+  git_root = git_root,
+  git_root_sync = void(git_root)
```

Comment:
This doesn't do anything. `void` doesn't *block* until the future completes, it just makes sure it's running to completion sometime in the future. Because of that, it's also not actually returning the result of `git_root` like that.

Reply:
Ah whoops.